Aterrizar/Aterrizar.Application.Tests/UserRepositoryStub.cs

```diff
@@ -0,0 +1,33 @@
+using Aterrizar.Application.Common.Interfaces.Persistance;
+using Aterrizar.Domain.Entities;
+
+namespace Aterrizar.Application.Tests;
+
+internal class UserRepositoryStub : IUserRepository
+{
+    private static readonly User? NULL_USER = null;
```

Comment:
Why static? I'm not saying is wrong, just wondering

Reply:
so you can't actually set any different value than null, I always want that to be null
![image](https://user-images.githubusercontent.com/18756969/178169800-f5631971-30b3-4043-89dd-6065b4046d0c.png)